docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
+- **<max_value>**: The maximum value in the range.
+
+Example
+~~~~~~~
+To create a discrete uniform distribution with possible values of 1, 2, 3, 4, and 5:
+::
 
-    \textit{min} + i * (max - min) / (nbins - 1)
+   VAR DUNIF 5 1 5
 
-for values of i between 0 and nbins-1.
+.. image:: fig/dunif.png
 
+Notes
+~~~~~
+Values are derived based on the formula:
+:math:`\text{min} + i \times (\text{max} - \text{min}) / (\text{nbins} - 1)`
+Where :math:`i` ranges from 0 to :math:`\text{nbins} - 1`.
 
-ERRF
-,,,,,
+ERRF: Error Function-Based Prior
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ERRF keyword is used to define a prior resulting from applying the error
-function to a normally distributed variable with mean 0 and variance 1. The
-keyword takes four arguments:
+The ``ERRF`` keyword allows creating prior distributions derived from applying the error function to a standard normal variable (with mean 0 and variance 1).
+This keyword offers flexibility in creating distributions of diverse shapes and symmetries.
 
+Syntax
+~~~~~~
 ::
 
   VAR8 ERRF MIN MAX SKEWNESS WIDTH
 
-The arguments MIN and MAX sets the minimum and maximum value of the transform.
-Zero SKEWNESS results in a symmetric distribution, whereas negative SKEWNESS
-will shift the distribution towards the left and positive SKEWNESS will shift it
-towards the right. Letting WIDTH be larger than one will cause the distribution
-to be unimodal, whereas WIDTH less than one will create a bi-modal distribution.
+Parameters
+~~~~~~~~~~
+- **MIN**: Specifies the minimum value of the transform.
+- **MAX**: Defines the maximum value of the transform.
+- **SKEWNESS**: Adjusts the asymmetry of the distribution.
+
+  - ``SKEWNESS < 0``: Shifts the distribution towards the left.
+  - ``SKEWNESS = 0``: Results in a symmetric distribution.
+  - ``SKEWNESS > 0``: Shifts the distribution towards the right.
+- **WIDTH**: Modulates the peakedness of the distribution.
+
+  - ``WIDTH = 1``: Generates a uniform distribution.
+  - ``WIDTH > 1``: Creates a unimodal, peaked distribution.
+  - ``WIDTH < 1``: Forms a bimodal distribution with peaks.
+
+Examples
+~~~~~~~~
+1. For a symmetric, uniform distribution:
+   ::
+   
+     VAR ERRF -1 1 0 1
+
+.. image:: fig/errf_symmetric_uniform.png
+
+2. For a right-skewed, unimodal distribution:
+   ::
+   
+     VAR ERRF -1 1 2 1.5
+
+.. image:: fig/errf_right_skewed_unimodal.png
+
+Notes
+~~~~~
+Keep in mind the interactions between the parameters, especially when both ``SKEWNESS`` and ``WIDTH`` are adjusted.
+Their combination can result in a wide range of distribution shapes.
+
+DERRF: Discrete Error Function-Based Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``DERRF`` keyword is a discrete version of the ``ERRF`` keyword.
+It is designed for creating distributions based on the error function but with discrete output values.
+This keyword facilitates the creation of distributions with various shapes and asymmetries.
```

Comment:
How about "... facilitates sampling from discrete distributions with various shapes and asymmetries"

Reply:
Fixed